index.src.html

```diff
@@ -850,6 +850,13 @@ <h3 id="determine-requests-referrer" dfn export>
       Let request's <dfn>referrerURL</dfn> be the result of <a href="#strip-url">stripping
       <var>referrerSource</var> for use as a referrer.</a>
     </li>
+    <li>
+      If the <a for=url>origin</a> of <a>referrerURL</a> has a <a for=origin>host</a> that
+      ends with <code>.onion</code> or <code>.onion.</code>, the special-use domain name defined
+      in [[!RFC7686]], and that <a for=url>origin</a> is not <a lt="same origin">the same</a> as
+      the <a for=url>origin</a> of <var>request</var>'s <a for=request>current URL</a>, then
```

Comment:
Could we just use the https://w3c.github.io/webappsec-referrer-policy/#cross-origin-referrer-request dfn here?

Reply:
Indeed. That reduces the unnecessary verbosity.